vg/_helpers.py

```diff
@@ -67,3 +67,9 @@ def _check_value_any(a, *shapes, name=None):
         raise ValueError(f"{preamble} with shape {shape_choices}; got None")
     else:
         raise ValueError(f"{preamble} with shape {shape_choices}; got {a.shape}")
+
+
+# adapted from https://stackoverflow.com/a/31465848
+def get_imported_names(module):
```

Comment:
Is the purpose of this function to work before and after the addition of `__all__` to one of these modules?

Reply:
yes, that was my thinking.

Reply:
I think it’s probably not worth it, I would just inline the bit of the if that you need in each test, and swap it out when you add `__all__` in the next PR.

Reply:
Alternatively it should go in a test helpers module which should get excluded at build time so it doesn’t get shipped with the library.

Reply:
done and done